index.md

```diff
@@ -4,6 +4,4 @@ layout: home
 nav_order: 1
 ---
 
-These are the GUAC Docs!
-
-![Under Construction](assets/images/under-construction.gif)
\ No newline at end of file
+Graph for Understanding Artifact Composition (GUAC) gives you organized and actionable insights into your software supply chain security position. GUAC ingests software security metadata, like SBOMs, and maps out the relationship between software. This allows you to see relationships between software so that you can fully understand your software security position. Using GUAC, you can drive higher-level organizational outcomes such as audit, policy, risk management, and even developer assistance.
```

Comment:
```suggestion
Graph for Understanding Artifact Composition (GUAC) gives you organized and actionable insights into your software supply chain security position. GUAC ingests software security metadata, like SBOMs, and maps out the relationship between software so that you can fully understand your software security position. Using GUAC, you can drive higher-level organizational outcomes such as audit, policy, risk management, and even developer assistance.
```